projects/driver_controls_pedal/inc/pc_cfg.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+// Pedal controls config file
+
+// #define PC_CFG_DEBUG_PRINT_EVENTS
+
+#define PC_CFG_CAN_DEVICE_ID SYSTEM_CAN_DEVICE_PEDAL_CONTROLS
+#define PC_CFG_CAN_BITRATE CAN_HW_BITRATE_500KBPS
+
+#define PC_CFG_CAN_RX \
+  { GPIO_PORT_A, 11 }
+#define PC_CFG_CAN_TX \
+  { GPIO_PORT_A, 12 }
+
+#define PC_CFG_PEDAL_ADC_RDY_PIN \
+  { GPIO_PORT_E, 2 }
```

Comment:
Where's Port E from? The LQFP48 only has ports A, B, C as far as I know.

Reply:
this is the adc ready pin, defined in the schematic https://github.com/uw-midsun/hardware/blob/master/MSXII_PedalBoard/Project%20Outputs%20for%20MSXII_PedalBoard/MSXII_PedalBoard_2.1.PDF

Reply:
If you look at the controller board, pin 15 on the mezzanine is PB2, not PE2. PE2 is not available in the STM32F072CBT6 variant. 

@aashmikamali let's fix the schematic and release a Rev 3.1 when you have time. This isn't a huge priority but should be done to avoid confusion in the future.